libs/native/src/main/java/org/elasticsearch/nativeaccess/LinuxNativeAccess.java

```diff
@@ -16,10 +16,20 @@ class LinuxNativeAccess extends PosixNativeAccess {
     Systemd systemd;
 
     LinuxNativeAccess(NativeLibraryProvider libraryProvider) {
-        super("Linux", libraryProvider);
+        super("Linux", libraryProvider, new PosixConstants(-1L, 9, 1));
```

Comment:
It's pretty readable in intellij (it shows the parameter name). I want to avoid creating individual static constants for each because they would just be passed through in this constants object, which is also statically held, meaning they would be in memory twice for no good reason.